back/app/policies/idea_policy.rb

```diff
@@ -58,9 +58,13 @@ def by_slug?
     show?
   end
 
+  def draft_by_phase?
+    show?
+  end
+
   def update?
-    return false if record.participation_method_on_creation.never_update?
-    return true if record.draft? || (user && UserRoleService.new.can_moderate_project?(record.project, user))
+    return false if !record.participation_method_on_creation.update_if_published? && !record.draft? && !record.will_be_published?
```

Comment:
Again, is it possible to update phygital surveys now?

Reply:
Should be. Will test though.

Reply:
Ah ok. I just thought it was not intended.